lib/OpenQA/Setup.pm

```diff
@@ -201,6 +201,9 @@ sub read_config {
         misc_limits => {
             untracked_assets_storage_duration => 14,
         },
+        'assets/storage_duration' => {
+            # Note: This empty section will not be used and is just here for completeness
```

Comment:
The section is actually used. There's just not a fix set of well-defined keys. Right?

Reply:
It's literally impossible to define a hash with no keys. It will not have any effect.

Reply:
In other words, I could also remove this, and it won't make a difference. This is only for the sake of having an overview of all the categories

Reply:
I think @Martchus's comment is more about the wording. I think `# intentionally left blank for overview` expresses your intention better :)